src/main/java/com/google/sps/servlets/AboutServlet.java

```diff
@@ -33,29 +35,42 @@
 
 @WebServlet(urlPatterns = URLPatterns.ABOUT)
 public class AboutServlet extends HttpServlet {
+  private String staticResponse;
 
   @Override
-  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println("REQUEST AT: " + request.getServletPath());
-    response.setContentType("text/html;");
-
+  public void init() {
     JinjavaConfig config = new JinjavaConfig();
     Jinjava jinjava = new Jinjava(config);
     try {
       jinjava.setResourceLocator(
           new FileLocator(
               new File(this.getClass().getResource(ResourceConstants.TEMPLATES).toURI())));
-    } catch (URISyntaxException e) {
-      System.err.println("templates dir not found!");
+    } catch (URISyntaxException | FileNotFoundException e) {
+      System.err.println(ErrorMessages.TEMPLATES_DIRECTORY_NOT_FOUND);
     }
 
     Map<String, Object> context = new HashMap<>();
     context.put(URLPatterns.URL, URLPatterns.ABOUT);
-    String template =
-        Resources.toString(
-            this.getClass().getResource(ResourceConstants.TEMPLATE_ABOUT), Charsets.UTF_8);
-    String renderedTemplate = jinjava.render(template, context);
 
-    response.getWriter().println(renderedTemplate);
+    try {
+      String template =
+          Resources.toString(
+              this.getClass().getResource(ResourceConstants.TEMPLATE_ABOUT), Charsets.UTF_8);
+      staticResponse = jinjava.render(template, context);
+    } catch (IOException e) {
+      System.err.println(ErrorMessages.templateFileNotFound(ResourceConstants.TEMPLATE_ABOUT));
+    }
+  }
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    response.setContentType("text/html;");
+
+    if (staticResponse == null) {
+      response.setStatus(500);
```

Comment:
5xx is typically for server errors. Doesn't this fit 404 more, Resource not found?